### DISCUSSION TITLE
Is this model supporting to run in the multithread env?

### DISCUSSION THREAD

**Discussion by lixikun** [snakers4/silero-vad#207] [Q&A] [answered]
when I run in multithread, I get errors as the pic show :

![image](https://user-images.githubusercontent.com/6120860/177981412-1644687b-897c-411b-948b-8d4d2f2cc8e7.png)

the code is 
![image](https://user-images.githubusercontent.com/6120860/177981829-5e4acbae-cdd2-44b8-b3f3-1b58741af34a.png)

**Comment by snakers4** [comment, marked as answer]
Hi,

Looks like you stumbled upon this - https://github.com/snakers4/silero-vad/issues#issuecomment-1050968656
Please confirm that this fix works.

**Reply by hamjam** [reply]
I have the same problem with @lixikun but I think the link you've mentioned, is corrupted.
Can you fix it?

**Reply by snakers4** [reply]
https://github.com/snakers4/silero-vad/issues/2#issuecomment-1050968656
